.github/ISSUE_TEMPLATE/release.md

```diff
@@ -1,59 +1,60 @@
 ---
-name: k6 Release 
-about: k6 release accommodates activities and a checklist that has the k6 open-source release process.
+name: k6 Release
+about: k6 release accommodates activities and a checklist with the k6 open-source release process.
 title: 'k6 release v0.4x.0'
+labels: ["release"]
 ---
 
-**Release Date**: 
+**Release Date**:
 
 25th May 2023 <<  TODO: WRITE HERE THE UPDATED RELEASE DATE
 
 ## Release Activities
 
 ### At the beginning of the cycle
 
-- [ ] Create a new branch `release-notes-v04x` and add a new related [release notes file](release-template/release%20notes) using the available [template](release-template/release%20notes/template.md).
-- [ ] Go through the potential [dependencies updates](Dependencies.md) then create a PR if any is relevant. 
+- [ ] Create a new `release-notes-v04x` branch and add a new release notes file using the available [template](release notes/template.md) to the [repository's `release notes` folder](/release notes).
+- [ ] Go through the potential [dependencies updates](Dependencies.md) and create a dedicated PR if any of them is relevant to this release.
```

Comment:
It could be here, or it could be in "one week before release" section, but it is worth mentioning that the release driver creates a branch in `grafana/k6-DefinitelyTyped`, and this branch should be used for opening the PR in the `DefinitelyTyped/DefinitelyTyped`.